godocgen_test.go

```diff
@@ -0,0 +1,160 @@
+package main
```

Comment:
Honestly, I think these tests are kind of ... not so useful. There are now 160 lines of code to test just a simple `if`:

		envPass := os.Getenv("GITHUB_PASS")
		if envPass != "" {
			c.Pass = envPass
		}

There should definitely be some tests for this program in general, but I don't know if this is worth it. Tests are code that need to be maintained, have bugs, can break, etc.

Reply:
Well, I do agree that those tests are not important, but they are not useless. In my POV, every code needs to be maintained, can break and can have bugs some day. 

But anyways, I guess that it's more about the company culture. So I'll leave only the [main test case](https://github.com/Teamwork/godocgen/pull/18/files#diff-23f896850032a318966cf7df4aae9c2aR40), or if you prefer, remove all the tests.